app/js/question_controller.js

```diff
@@ -414,6 +415,13 @@
       question_form = document.getElementById('question_form');
       question_form.addEventListener('submit', submit_comment);
 
+      question_submit_button =
+        document.getElementById('question_submit_button');
+      question_submit_button.addEventListener('click', function(evt) {
+        evt.preventDefault();
+        question_form.dispatchEvent(new Event('submit'));
+      });
```

Comment:
oh oops. i tried switching to an `<a>` tag intead of `<button>` and I needed this but that was stupid and unnecessary. i'll ditch this code.